keras_cv/backend/__init__.py

```diff
@@ -58,10 +58,15 @@
                 src_attr, dst_attr = attr, attr
             attr_val = getattr(src_mod, src_attr)
             setattr(dst_mod, dst_attr, attr_val)
+
+    keras_core_backend = keras.backend.backend
 else:
     from tensorflow import keras
 
     # TF Keras doesn't have this rename.
     keras.activations.silu = keras.activations.swish
 
+    # None => indicator that multi-backend is turned off.
+    keras_core_backend = lambda *a, **kw: None
```

Comment:
Let's use `from keras_cv.backend.config import multi_backend` to check whether we're multi-backend